src/bindings/c/src/ov_core.cpp

```diff
@@ -318,3 +329,90 @@ void ov_core_versions_free(ov_core_version_list_t* versions) {
         delete[] versions->versions;
     versions->versions = nullptr;
 }
+
+#ifdef OPENVINO_ENABLE_UNICODE_PATH_SUPPORT
+inline std::string wstring_to_string(const std::wstring& wstr) {
```

Comment:
I think we shouldn't duplicate this function in C API.

You can link C API with `openvino::util` in order to use existed function.

Reply:
done